### ISSUE TITLE
Showing related items on data page

### ISSUE THREAD

**Issue by jacobgibbs** [avillafiorita/jekyll-datapage_gen#18]
Firstly, thank you for making this plugin! It is extremely useful.

I am wondering if it is possible to use the HTML template for the page to be able to pull in links to 3 other related pages from the same data file? 
Within my .yml file I have a `category` attribute, so I was hoping to use the category of the current page to pull in 'You may also like...' suggestions for other pages with the same category.

**Comment by avillafiorita** [comment]
Thanks for your question and sorry for taking so long to get back to you.

From what I understand: 

1. The CSV files contains the specification of various pages to generate. 
2. You are using my plugin to generate the pages, but---and this is the critical point---when you are generating a page, you would like to read the same CSV file looking for other records

I am not sure it workds, but you could be able to achieve your goal using the standard machinery Jekyll provides for reading CSV files. Can't you add the liquid markup described [here](https://jekyllrb.com/docs/variables/) to the template used to generate the pages to access the data you need?

In other words, what I am suggesting is that you use the plugin to read your CSV file one record at a time, but you use Liquid markup in the template used to generate the pages to read the same CSV file looking for the related data you need.
